src/Model/Contact.php

```diff
@@ -1152,7 +1152,7 @@ public static function createFromProbe($uid, $url, $interactive = false, $networ
 		Addon::callHooks('follow', $arr);
 
 		if (empty($arr)) {
-			$result['message'] = L10n::t('Contact cannot be added.');
+			$result['message'] = L10n::t('The contact could not be added. It is possible you are attempting to add a contact that requires an Addon that is not enabled on this system.');
```

Comment:
The addon was enabled, but the user didn't configure it. If it wasn't enabled the hook wouldn't be called at all.

Reply:
Changed again :-)